pkg/get/download.go

```diff
@@ -119,6 +98,39 @@ func Download(tool *Tool, arch, operatingSystem, version string, downloadMode in
 	return outFilePath, finalName, nil
 }
 
+func downloadFile(downloadURL string, displayProgress bool) (string, error) {
```

Comment:
I'm a bit conflicted about leaking the progress bar into `downloadFile`, but let's go with it for the time being.